docs/usage/self-hosted-experimental.md

```diff
@@ -35,6 +35,10 @@ If set to any value, Renovate will always paginate requests to GitHub fully, ins
 
 If set to "false" (string), Renovate will remove any existing `package-lock.json` before attempting to update it.
 
+## RENOVATE_TERRAFORM_LOCK_FILE
```

Comment:
```suggestion
## RENOVATE_X_TERRAFORM_LOCK_FILE
```
Please use `RENOVATE_X_` prefix for experimental options, as that is the new standard prefix.